lib/muffin_man/listings/v20200901.rb

```diff
@@ -0,0 +1,45 @@
+module MuffinMan
+  module Listings
+    class V20200901 < SpApiClient
+      REQUIREMENTS = %w[LISTING LISTING_PRODUCT_ONLY LISTING_OFFER_ONLY].freeze
+      REQUIREMENTS_ENFORCED = %w[ENFORCED NOT_ENFORCED].freeze
+      LOCALE = %w[DEFAULT ar ar_AE de de_DE en en_AE en_AU en_CA en_GB en_IN en_SG en_US es es_ES es_MX es_US fr
+                  fr_CA fr_FR it it_IT ja ja_JP nl nl_NL pl pl_PL pt pt_BR pt_PT sv sv_SE tr tr_TR zh zh_CN zh_TW]
+               .freeze
+
+      def search_definitions_product_types(marketplace_ids, keywords = nil)
+        @local_var_path = "/definitions/2020-09-01/productTypes"
```

Comment:
Can we constantize the shared part of the path?

Reply:
Does the 2020-09-01 reflect a particular version of the API?

Reply:
> Does the 2020-09-01 reflect a particular version of the API?

yes

Reply:
Ref: https://developer-docs.amazon.com/sp-api/docs/product-type-api-use-case-guide